birdhouse/optional-components/README.md

```diff
@@ -112,3 +112,26 @@ Magpie will be automatically configured to give complete public anonymous
 access for this WPS service.
 
 Canarie monitoring will also be automatically configured for this WPS service.
+
+
+## All access given to test user
+
+Can be used for testing purposes.
```

Comment:
suggested text change "By enabling this component, all WPS services and data on Thredds are completely public, please beware.  Once enabled, if you need to revert the change, you have to do it manually by logging into Magpie.  Just disabling this component will not revert the change."